src/chocolatey/infrastructure.app/nuget/ChocolateyNugetCredentialProvider.cs

```diff
@@ -50,6 +73,12 @@ public ICredentials GetCredentials(Uri uri, IWebProxy proxy, CredentialType cred
                 this.Log().Warn("Invalid credentials specified.");
             }
 
+            Uri resolveUri;
+            if (_resolveDownloadUrls.TryGetValue(uri, out resolveUri))

```

Comment:
For consistency there should probably be a newline here.
```suggestion
            Uri resolveUri;
            
            if (_resolveDownloadUrls.TryGetValue(uri, out resolveUri))
```

Also: is this going to result in a warning about the variable not being assigned a value?

Reply:
> For consistency there should probably be a newline here.

CLI 1.x do not have that consistency, which is why I didn't care about it. I don't mind either way though.

> Also: is this going to result in a warning about the variable not being assigned a value?

No, it won't. It is not used anywhere outside of where it is set.